### ISSUE TITLE
not relaunching after update

### ISSUE THREAD

**Issue by zoulztealer** [madbomb122/BlackViperScript#32]
**Question**   
a little bit.

**Describe the bug**   
tool doesnt relaunch after update.

```
Displaying GUI Now

To exit you can close the GUI or PowerShell Window.
|----------------------------------------------------|
|                                                    |
|                   Update Found!                    |
|                                                    |
| Downloading version 5.3.1.                         |
| Will run BlackViper-Win10.ps1                      |
| after download is complete.                        |
|                                                    |
|----------------------------------------------------|
PS C:\Users\intel\Desktop\BlackViperScript-master>
```

**To Reproduce**   
1. update from previous to latest version

**Expected behavior**   
tool should relaunch after in-app update.

**Additional context**
 - Script Version: 5.3.0 to 5.3.1 / 4.2
 - Release Type: stable

**Comment by zoulztealer** [comment]
launching over bat file. sry, didnt read the note on 5.2.4.

**Comment by madbomb122** [comment]
it reopens for me.. it should open in a new window

**Comment by madbomb122** [comment]
btw, did you get the updated `update.bat` file?

also that problem that you had with the **all** setting not being remember should be fixed if you got the update` _Win10-BlackViper.bat` file, if you havnt gotten it.

**Comment by madbomb122** [comment]
another question.. did the script get updated?

**Comment by zoulztealer** [comment]
the in-app update worked in all previous versions. i see all switch has been fixed, too, very comfortable now for me, haha. will check update now again from 5.2.1.

when running in-app update for 5.2.1 from powershell admin console via ps script or bat file i get this:

```
> |----------------------------------------------------|
> |                   Update Found!                    |
> |----------------------------------------------------|
> |                                                    |
> | Downloading version 5.3.2.                         |
> | Will run BlackViper-Win10.ps1                      |
> | after download is complete.                        |
> |                                                    |
> |----------------------------------------------------|
> Der Befehl "C:\Users\intel\Downloads\BlackViperScript-master" ist entweder falsch geschrieben oder
> konnte nicht gefunden werden.
```


looks to me like theres something wrong with paths or a command ran from wrong path / code typo.

when i run update.bat:

```
> PS C:\Users\intel\Downloads\BlackViperScript-master (3)\BlackViperScript-master> .\Update.bat
> "\BlackViperScript-master\BlackViper-Win10.ps1" kann syntaktisch an dieser Stelle nicht verarbeitet werden.
```


regards and thank you very much for the nice work

**Comment by madbomb122** [comment]
i have no idea what is happening, i downloaded the script right from github.
changed the version number in the file (so it will update) and it works fine for me.

i wonder if it's because of the non-english os.

**Comment by madbomb122** [comment]
i didnt notice the message about the update.bat.. i duplicated your folder name.. and tried doing an update and it didnt work.. at least i know were to start now.

for now rename your folder to remove the (3).. that is causing the issue

**Comment by madbomb122** [comment]
issue should be solved in  v5.3.5
However you will need to manually update if you cant update using the script
